llvm/lib/TableGen/Record.cpp

```diff
@@ -2264,8 +2264,8 @@ DefInit *VarDefInit::instantiate() {
   if (!Def) {
     RecordKeeper &Records = Class->getRecords();
     auto NewRecOwner = std::make_unique<Record>(Records.getNewAnonymousName(),
-                                           Class->getLoc(), Records,
-                                           /*IsAnonymous=*/true);
+                                                Class->getLoc(), Records,
+                                                /*Kind=*/Record::RK_Anonymous);
```

Comment:
Now that the argument is no longer a boolean, `/*Kind=*/` is no longer needed here and in other places.